css-positioning/relative.css

```diff
@@ -0,0 +1,47 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+  display: flex;
+}
+
+.pokemon > img {
+  width: 150px;
+  flex-basis: 20%;
+
```

Comment:
You do not need the `flex-basis` here, just the `width` that was provided is good enough.